internal/processor/parser/app.go

```diff
@@ -143,19 +135,23 @@ func (am *AppParser) updateCacheLoop() {
 func (am *AppParser) warmupCache() {
 	am.log.Infof("Warming up cache...")
 
-	apps, err := listApps(am.CFClient, am.cacheWorkers, am.log)
+	cfapps, err := am.cfClient.GetApplications()
 	if err != nil {
-		am.log.Errorf("Error warming up cache, couldn't get list of apps: %v", err)
+		am.log.Errorf("error warming up cache, couldn't get list of apps: %v", err)
 		return
 	}
-
-	for _, resolvedApp := range apps {
-		am.AppCache.Add(resolvedApp)
+	for _, cfapp := range cfapps {
+		_, err := am.AppCache.Add(cfapp)
+		if err != nil {
+			fmt.Println(err)
+			am.log.Errorf("an error occurred when adding app to the cache: %v", err)
+			return
```

Comment:
I don't think this should be a hard fail. We should just skip this app, and continue

Reply:
Done.